docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -0,0 +1,47 @@
+General configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``case-insensitive-name-matching``
+    - Support case insensitive database and collection names
```

Comment:
"Enable support for..."

Reply:
case-insensitive

Reply:
Pre-existing. Will change in separate PR.